src/main/java/org/spongepowered/api/command/spec/CommandSpec.java

```diff
@@ -383,8 +392,9 @@ public Text getUsage(CommandSource source) {
             builder.append(desc.get(), Text.NEW_LINE);
         }
         builder.append(getUsage(source));
-        if (this.extendedDescription != null) {
-            builder.append(Text.NEW_LINE, this.extendedDescription);
+        Optional<Text> extdesc = getExtendedDescription(source);
+        if (extdesc.isPresent()) {
+            builder.append(Text.NEW_LINE, extdesc.get());
```

Comment:
`this.getExtendedDescription(source).ifPresent(builder::append);` or so, instead of these 3 lines